Cargo.toml

```diff
@@ -55,14 +55,14 @@ async-trait = "0.1.86"
 aws-config = "1"
 aws-sdk-glue = "1.39"
 bimap = "0.6"
-bitvec = "1.0.1"
```

Comment:
Thank for this change, but `funty` is maintained by the same team of `bitvec` and is not updated for 2 years.

Reply:
removed